table/block_based/block_based_table_reader.cc

```diff
@@ -2153,10 +2226,13 @@ Status BlockBasedTable::MaybeReadBlockAndLoadToCache(
     s = GetDataBlockFromCache(key, ckey, block_cache, block_cache_compressed,
                               ro, block_entry, uncompression_dict, block_type,
                               get_context);
-
+    if (block_entry->GetValue()) {
```

Comment:
This condition would hold if the block was found in either the uncompressed or the compressed block cache. Just wanted to confirm if that's the definition of 'cache hit' you're going for.

Reply:
We are focusing on uncompressed block cache right now. I added a TODO here.